sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -386,6 +391,9 @@ static void _send_unicast(kernel_pid_t iface, uint8_t *dst_l2addr,
 
     DEBUG("ipv6: send unicast over interface %" PRIkernel_pid "\n", iface);
     /* and send to interface */
+#ifdef MODULE_NETSTATS_IPV6
+    gnrc_ipv6_netif_get_stats(iface)->tx_unicast_count++;
```

Comment:
I see, the `iface` parameter breaks this idea. But I still think this should be somehow cached .. most of the time we only have one interface.